scripts/mkcloud

```diff
@@ -519,6 +523,23 @@ EOS
     echo "you can now proceed with installing crowbar"
 }
 
+function createadminsnapshot()
+{
+    virsh shutdown $cloud-admin
+    wait_for 150 1 "virsh domstate $cloud-admin|grep shut.off" 'admin node to shut down'
+    lvremove -f $admin_node_disk.snap
+    safely lvcreate -l100%ORIGIN -s -n $cloud.admin.snap $admin_node_disk
+    virsh start $cloud-admin
+    wait_for_crowbar_ssh
+}
+
+function restoreadminfromsnapshot()
+{
+    virsh destroy $cloud-admin
```

Comment:
Is missing some kind of guard to make sure that is safe to `destroy` because there is a previous snapshot?